assets/js/base/hooks/cart/use-store-cart.js

```diff
@@ -35,21 +38,40 @@ const defaultCartData = {
  * @return {StoreCart} Object containing cart data.
  */
 export const useStoreCart = ( options = { shouldSelect: true } ) => {
+	const { isEditor } = useCartContext();
 	const { shouldSelect } = options;
 
 	const results = useSelect(
 		( select ) => {
 			if ( ! shouldSelect ) {
 				return null;
 			}
+
+			if ( isEditor ) {
+				// Convert keys to camelCase like the reducer in the cart data store.
+				const cartData = mapKeys( previewCart, ( _, key ) =>
+					camelCase( key )
+				);
```

Comment:
This is kind of unnecessary here since you're directly assigning the values in the returned object anyways.

Reply:
Ah yes, I can handle this below.